### DISCUSSION TITLE
FTP(S) downloads with segmented downloads fail

### DISCUSSION THREAD

**Discussion by tatze96** [iterate-ch/cyberduck#15882] [Q&A]
Downloads fail with a connection error, although the server log does not reflect the lost connection while segmented downloads are activated. The download works correctly when performed with deactivated segmented downloads.

The used server doesn't have a concurrent connection limit, so that can't be the culprit in this case.

- operating system: macOS
- Cyberduck version: 8.7.2
- tested protocol: FTPS

**Comment by dkocher** [comment]
Cannot reproduce with a download of a ~150MB file from a `vsftpd` server (version 2.2.2) with _Transfers → Connections_ set to 20 connections [^1]. 

[^1]: https://docs.cyberduck.io/cyberduck/transfer/#limit-number-of-parallel-connections
